script/setup

```diff
@@ -22,6 +22,7 @@ echo "Cleaning out old logs"
 echo "Installing libraries and plugins"
 { gem list -i bundler || gem install bundler
   bundle
+  script/install_toxiproxy.sh
```

Comment:
If you're making it a hard and not optional dependency here, why even have the `skip`?